db/migrate/20250121163914_drop_claims_tables.rb

```
@@ -0,0 +1,13 @@
+class DropClaimsTables < ActiveRecord::Migration[7.0]
```

rubocop (1.69.1)

[convention] Missing top-level documentation comment for `class DropClaimsTables`.

(Style/Documentation)